sop/modules/OSD/pages/CephMonQuorumAtRisk.adoc

```diff
@@ -28,7 +28,6 @@ include::partial$prerequisites.adoc[]
 
 ==  Alert
 
-WIP: Needs procedure for quorum recovery.
```

Comment:
When we are removing this WIP - shouldn't we add the actual procedure for quorum recovery here?

Reply:
This was a mistake, i think the steps for quorum recovery still needs to be researched, let me check, i will update it in the next revision